### DISCUSSION TITLE
I found public demo api is not working from today (or maybe few days ago)

### DISCUSSION THREAD

**Discussion by sukam09** [realworld-apps/realworld#960] [DEMO / API health status] [answered]
Hello. I tried to test post method in my realworld-example-app using public demo api [here](https://realworld-docs.netlify.app/docs/specs/frontend-specs/api#demo-api) but it doesn't work from some day.

I wonder because I remember that I checked the api and its [documentation](https://api.realworld.io/api-docs) until recently but now I can only see the Heroku Application Error saying that "There's nothing here, yet." so I want to know about this.

Moreover, I failed to create new server in pgAdmin 4 (maybe will be able to fix later) so I deployed using [Heroku App](https://realworld-example-app-api.herokuapp.com/), however I couldn't find endpoints and what I found was "404 Error" only. Therefore, I need your help or some feedbacks.

Please check these issues and I'm looking forward to getting your answer.

Thank you.

**Comment by geromegrignon** [comment]
Hi @sukam09,

Thanks for your feedback. I confirm the current situation.
I'll review it today to fix it.

**Comment by geromegrignon** [comment, marked as answer]
It should be fine by now. Sorry for that.

**Reply by sukam09** [reply]
I checked it works well now. Thanks :)
